public/assets/css/module.css

```diff
@@ -1,16 +1,15 @@
-
 .code-area {
   background-color: #333333;
   color: #fff;
+  height: 77% !important;
```

Comment:
We do it to override bootstrap class property. How can we override it with another selector?